trunk/SUAVE/Methods/Aerodynamics/AVL/untrimmed.py

```diff
@@ -0,0 +1,27 @@
+# untrimmed.py
+#
+# Created:  Jan 2014, T. Orra (base file in low fidelity)
+# Modified: May 2017, T. MacDonald  
+
+def untrimmed(state,settings,geometry):
+
+    # Unpack inputs
+    conditions     = state.conditions
+    configuration  = settings
+    drag_breakdown = conditions.aerodynamics.drag_breakdown
+
+    # Various drag components
+    parasite_total        = conditions.aerodynamics.drag_breakdown.parasite.total            
+    induced_total         = conditions.aerodynamics.drag_breakdown.induced.total  
+    invisid_total         = parasite_total + induced_total
```

Comment:
Fixed. However I discovered that the AVL analysis was not actually being used by AVL test. I'm going to finish these fixes and then I'll handle that.